doc/sedcli.8

```diff
@@ -81,6 +81,15 @@ Changes the lock state for the device. Following access modes are available:
 .IP "\fB\-P -d <device>\fR or \fB\-\-set-password --device <device>\fR"
 Updates password for Admin1 authority in Locking SP.
 
+.IP "\fB\-M -d <device> -e <enable> -m <done>\fR or \fB\-\-mbr-control --device <device> --enable {TRUE|FALSE} --done {TRUE|FALSE}\fR"
+.IP
+Enable/Disable MBR Shadow and Set/Unset MBR Done.
+MBR Done is TRUE only when the MBR Shadow is Enabled.
+
+.IP "\fB\-W -d <device> -f <File path containing Pre-boot Application(PBA) image> [-o]\fR or \fB\-\-write-mbr --device <device> --file <File path containing Pre-boot Application(PBA) image> [--offset]\fR"
+Write data into MBR shadow region(MBR table).
+Enable MBR shadow, for the PBA to take effect.
```

Comment:
I don't think it actually enables MBR shadow. it just writes the data into MBR table. enabling is done via --mbr-control command

Reply:
What i was trying to convey here is that, the user should have enabled the MBR shadow in-order to let the OS use the data stored in MBR table... maybe I can rephrase the sentence.

Reply:
yep I see now.